README.md

```diff
@@ -12,38 +11,35 @@ Before starting the practical, make sure you have the following ready, if not, p
 * The URL [link](https://learning.ds-wizard.org/) for DSW training instance.
 * The [example scenario](https://github.com/elixir-luxembourg/DS-DM-training/blob/master/resources/DM-DP_RunningExample.pdf) for our workshop.
 
-## Exercice 1 : sign up
+## Exercice 1: sign up
 
 * Go to the url [link](https://learning.ds-wizard.org/) for DSW training and `Sign up`.
 
-
 ## Exercise 2: Create questionnaire (project) from existing knowledge model
 
 In the DSW tool, _Knowledge Models_ act as source databases that hold requirements in one or more sub-area of RDM.  
 The tool comes pre-packed with a comprehensive default knowledge model. In this practical we will use a cut-down version of this model. To create and fill in a questionnaire:
 
-
 1. Login to the DSW.<br/>
-2. On the left-hand side navigation menu click `Knowledge Models` to list models available. You should see `DM-DS Workshop Model` as below: <br/> <br/><br/>![Alt](images/list_workshop_km.png)<br/><br/><br/>
+2. On the left-hand side navigation menu click `Knowledge Models` to list models available. <br/> <br/><br/>![Alt](images/list_workshop_km.png)<br/><br/><br/>
 3. Click on the knowledge model named `DM-DS_Workshop_Model` to go to its overview page, which displays model's version history and available actions.<br/>
 4. Click `Create Project` on top right to create a project as a questionnaire from the workshop model: <br/><br/><br/>![Alt](images/project.png)<br/><br/><br/>
```

Comment:
I find it a bit inconsistent - is  it project or questionnaire? Can I create project as something else than questionnaire? These two terms and their relationship might be explained or this sentence rephrased

Reply:
For the question of the name 'questionnaire' or 'project', before it was questionnaire the term used in the tool now it's project, but it's still a questionnaire. That's why I put create project as a questionnaire to be able to keep the term questionnaire.

Reply:
I should explain it clearly on the presentation.